src/pepr/operator/crd/validators/package-validator.ts

```diff
@@ -92,6 +92,27 @@ export async function validator(req: PeprValidateRequest<UDSPackage>) {
         `The client ID "${client.clientId}" uses an invalid secret name ${client.secretName}`,
       );
     }
+    // If standardFlowEnabled is undefined (defaults to `true`) or explicitly true and there are no redirectUris set, deny the req
+    if (client.standardFlowEnabled !== false && !client.redirectUris) {
+      return req.Deny(
+        `The client ID "${client.clientId}" must specify redirectUris if standardFlowEnabled is turned on`,
```

Comment:
nit: Will this message confuse someone (probably most), who will not know they have `standardFlowEnabled:  true` because they did not explicitly set it to `true`?

Reply:
added a default note